package-lock.json

```diff
@@ -8,11 +8,9 @@
       "license": "Apache-2.0",
       "workspaces": [
         "cdp-agentkit-core",
-        "cdp-langchain"
+        "cdp-langchain",
+        "cdp-langchain/examples/chatbot"
```

Comment:
we don't want the examples to be a workspace. These should be stand alone so that developers can easily strip them out of the monorepo to use as quick start foundations for their projects